molequeue/app/testing/scripts/sendRpcKill.py

```diff
@@ -10,7 +10,7 @@
 def run_test():
   global socketname
   if debug:
-    print "Connecting to socket: %s"%socketname
+    print ("Connecting to socket: %s"%socketname)
```

Comment:
Correct.  `print()` is a function in Python 3 but write it like other functions as `print()`, not `print ()`...
```suggestion
    print("Connecting to socket: %s"%socketname)
```

Reply:
> Correct. `print()` is a function in Python 3 but write it like other functions as `print()`, not `print ()`...

Yeah sure, that may have happened due to auto formatting of code

Reply:
I see no such automation...  https://github.com/OpenChemistry/molequeue/actions

Reply:
> I see no such automation... https://github.com/OpenChemistry/molequeue/actions

Sorry I meant by VS code in my pc.